lnst/Recipes/ENRT/MPTCPRecipe.py

```diff
@@ -83,13 +97,19 @@ def test_wide_configuration(self):
         if "ipv4" in self.params.ip_versions:
             host1.mptcp.add_endpoints(host1.eth1.ips_filter(family=AF_INET), flags=MPTCPFlags.MPTCP_PM_ADDR_FLAG_SUBFLOW)
             # Need route on client side to populate forwarding table
-            host1.run(f"ip route add 192.168.101.0/24 dev {host1.eth1.name} via 192.168.102.2 prio 10000")
+            host1.run(
+                f"ip route add {self.params.net1_ipv4} dev {host1.eth1.name}"
+                f" via {save_addrs[host2.eth1][AF_INET]} prio 10000"
+            )
             # Need to disable rp_filter on server side
             host2.run("sysctl -w net.ipv4.conf.all.rp_filter=0")
 
         if "ipv6" in self.params.ip_versions:
             host1.mptcp.add_endpoints(host1.eth1.ips_filter(family=AF_INET6), flags=MPTCPFlags.MPTCP_PM_ADDR_FLAG_SUBFLOW)
-            host1.run(f"ip route add fc00::/64 dev {host1.eth1.name} via fc01::2 prio 10000")
+            host1.run(
+                f"ip route add {self.params.net1_ipv6} dev {host1.eth1.name}"
+                f" via {save_addrs[host2.eth1][AF_INET6]} prio 10000"
+            )
```

Comment:
hm, looking at this now, is this supposed to only be configured in one way? shouldn't a similar route be added also on `host2` towards `host1`?

this isn't really related to the ip networks work, but you creating the `save_addrs` dictionary sort of highlighted the asymmetry to me.

Reply:
Yeah im pretty sure its correct, the asymetry is due to the direction of traffic. If you want i can look at this later when i have some time to get back into mptcp headspace.

Reply:
discussed on irc, this should be ok.